src/PlaywrightSharp/PlaywrightSharp.csproj

```diff
@@ -18,7 +18,7 @@
     <ReleaseVersion>0.0.0</ReleaseVersion>
     <BuildFromSource>True</BuildFromSource>
     <AnalysisMode>AllEnabledByDefault</AnalysisMode>
-    <NoWarn>1701;1702;CS0067</NoWarn>
+    <NoWarn>1701;1702;CS0067;1734;1574</NoWarn>
```

Comment:
What are these?

Reply:
1734 is stuff like this:
`XML comment on 'IDownload' has a paramref tag for 'acceptDownloads', but there is no parameter by that name (CS1734) (PlaywrightSharp)`
 `/// Browser context **must** be created with the <paramref name="acceptDownloads"/>`
I created [this issue](https://github.com/microsoft/playwright-sharp/issues/1304) to fix that.

I removed `1574`.